packages/config-eslint/package.json

```
@@ -17,18 +17,22 @@
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "check-rules": "../../node_modules/.bin/ts-node tools/check-rules/index.ts"
```

Do you need this? Npm scripts have access to the bins.
```suggestion
    "check-rules": "ts-node tools/check-rules/index.ts"
```

---

For some reason, in this repo, npm nor yarn add the "root" node_modules to PATH 🤷‍♂️ You can try it yourself, if you want, maybe it's just problem on my machine.

---

Now it's working 🤦‍♂️ Weird.